README.md

```diff
@@ -58,6 +58,18 @@ $ npm install --production
 $ node . --notebooks path/to/notebooks
 ```
 
+## Install and run from source as an electron app
+
+```bash
+# Install
+$ git clone https://github.com/netgusto/nodebook
+$ cd nodebook
+$ npm install --production
```

Comment:
I tested with `install --production`, and electron cannot launch (it is not installed because it's listed as a dev dep).

I think this line should be `$ npm install` (that is, without `--production`).